packages/react-utils/package.json

```diff
@@ -39,6 +39,7 @@
     "@onaio/session-reducer": "^0.0.12",
     "@opensrp/notifications": "^0.0.2",
     "@opensrp/server-service": "^0.0.13",
+    "@opensrp/product-catalogue": "0.0.2",
```

Comment:
what necessiated this change

Reply:
I was using it at some point....I will remove it

Reply:
@p-netm Yes I am still using it for `loadProductCatalogue`

Reply:
how? are you calling `loadProductCatalogue` from 'react-utils`

Reply:
@p-netm I am using it [here](https://github.com/OpenSRP/web/blob/311-add-edit-inventory/packages/inventory/src/containers/InventoryAddEdit/index.tsx#L126)

Reply:
@kelvin-muchiri I still fail to see why we need to install product-catalogue package in *react-utils*, seems like link you mentioned above points to a function that is declared and used in the inventory module.
Maybe you meant to install the product-catalogue package in the inventory package

Reply:
@p-netm Oh yes I meant to install it in the inventory package